freud/diffraction.pyx

```diff
@@ -263,6 +265,277 @@ cdef class StaticStructureFactorDebye(_Compute):
             return None
 
 
+cdef class StaticStructureFactorDirect(_Compute):
```

Comment:
Since this class exposes the same interface as `StaticStructureFactorDebye`, it may be worth having a base class (at least on the python side) so we only have to write the code for common properties once.

Reply:
Is there a good example of this already in freud? Would this class be visible to users and on docs? What kind of documentation would we have for that class?

Reply:
Have a look at the PMFT module, each PMF class inherits from `_PMFT`. In the case of the PMFT module, there is also a base class on the cpp side, and after looking at the static structure factor cpp classes, I think we should have a base class for those also.

Reply:
I'll check. This might take a while, especially for cpp side.